package/MDAnalysis/core/groups.py

```diff
@@ -3111,6 +3111,26 @@ def select_atoms(self, sel, *othersel, periodic=True, rtol=1e-05,
                 radius 5, external radius 10 centered on the COG. In z, the
                 cylinder extends from 10 above the COG to 8 below. Positive
                 values for *zMin*, or negative ones for *zMax*, are allowed.
+            box *dimensions* *dN_min* *dN_max* *selection*
```

Comment:
```suggestion
            box *dimensions* *dN_min* *dN_max* [*dN_min* *dN_max*] [*dN_min* *dN_max*] *selection*
```
Specify also the optional arguments (depending on `dimension`)?